mmdet/models/detectors/base.py

```diff
@@ -1,16 +1,16 @@
-import warnings
-from abc import ABCMeta, abstractmethod
-from collections import OrderedDict
-
 import mmcv
 import numpy as np
-
 import pycocotools.mask as maskUtils
 import torch
 import torch.distributed as dist
-
 import torch.nn as nn
+import warnings
+from abc import ABCMeta, abstractmethod
+from collections import OrderedDict
+from contextlib import contextmanager
+from functools import partial
 from mmcv.utils import print_log
+from nncf.dynamic_graph.context import no_nncf_trace
```

Comment:
import error in case of not installed nncv

Reply:
Fixed